front/app/components/ProjectAndFolderCards/components/Topbar/Tabs.tsx

```diff
@@ -85,40 +85,44 @@ const Tabs = ({
   availableTabs,
   onChangeTab,
 }: Props) => {
-  const tabsRef = useRef({});
+  const tabsRef = useRef<HTMLButtonElement[]>([]);
 
   const handleClickTab = (tab: PublicationTab) => () => {
     if (currentTab === tab) return;
     onChangeTab(tab);
   };
 
   const handleKeyDownTab = ({ key }: KeyboardEvent<HTMLButtonElement>) => {
-    if (!LEFT_RIGHT_ARROW_KEYS.has(key)) return;
-
-    const selectedTab = getSelectedTab(currentTab, availableTabs, key);
-    onChangeTab(selectedTab);
-    tabsRef.current[selectedTab].focus();
+    const arrowLeftPressed = key === 'ArrowLeft';
+    const arrowRightPressed = key === 'ArrowRight';
+
+    if (arrowLeftPressed || arrowRightPressed) {
+      const nextTabIndex = getSelectedTabIndex(
+        currentTab,
+        availableTabs,
+        arrowLeftPressed ? 'ArrowLeft' : 'ArrowRight'
+      );
+
+      onChangeTab(availableTabs[nextTabIndex]);
+      tabsRef.current[nextTabIndex].focus();
+    }
   };
 
   return (
     <TabsContainer role="tablist">
-      <ScreenReaderOnly>
-        <FormattedMessage {...messages.a11y_publicationStatusTabs} />
-      </ScreenReaderOnly>
```

Comment:
Not necessary, the screen reader clearly pronounces we're in a tab system. You can (should) download NVDA to test :-)